src/js/analyses/components/Create/WorkflowSelector.js

```diff
@@ -17,24 +92,45 @@ export const getCompatibleWorkflows = (dataType, hasHmm) => {
 };
 
 export const WorkflowSelector = ({ dataType, hasError, hasHmm, workflows, onSelect }) => {
```

Comment:
```suggestion
export const WorkflowSelector = ({ dataType, hasError, hasHmm, selectedWorkflows, onSelect }) => {
```
(or other name that makes it obvious that these are the selected workflows rather than just a generic list of workflows)